deli_counter.rb

```diff
@@ -0,0 +1,46 @@
+# Anshul Arora
+katz_deli = []
+$current_line = []
+
+def line(arr)
+  if arr.empty?
+    puts "The line is currently empty." 
+  else
+    $current_line.clear
+    line_helper(arr)
+    puts "The line is currently:#{$current_line.join}"
+  end
+end
+
+
+def line_helper(arr)
+  arr.each do |name|
+    position = arr.index(name) + 1
+    pos = " " + position.to_s + ". "
+    $current_line.push(pos, name)
+  end
+end
+
+
+def take_a_number(katz_deli, name)
+  katz_deli.push(name)
```

Comment:
fun fact `<<` is shorthand for `.push` i usually would write this line as `katz_deli << name`